### DISCUSSION TITLE
python工作流参数流转

### DISCUSSION THREAD

**Discussion by trustTiMo** [apache/dolphinscheduler#12203] [General]
python中上游工作流定义个参数怎样传到下游？

**Comment by DarkAssassinator** [comment]
Hi @trustTiMo Please decribe your question in English.
Now DS just support Shell、SQL、Procedure tasks, please check the doc: https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/guide/parameter/context.html
